server/telemetry.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"github.com/mattermost/mattermost/server/public/pluginapi"
+	"github.com/mattermost/mattermost/server/public/pluginapi/experimental/bot/logger"
+	"github.com/mattermost/mattermost/server/public/pluginapi/experimental/telemetry"
+
+	"github.com/mattermost/mattermost-plugin-confluence/server/config"
+)
+
+const (
+	keysPerPage           = 1000
+	ConfluenceUserInfoKey = "_userinfo"
+)
+
+func (p *Plugin) TrackEvent(event string, properties map[string]interface{}) {
+	err := p.tracker.TrackEvent(event, properties)
+	if err != nil {
+		p.API.LogDebug("Error sending telemetry event", "event", event, "error", err.Error())
+	}
+}
+
+func (p *Plugin) TrackUserEvent(event, userID string, properties map[string]interface{}) {
+	err := p.tracker.TrackUserEvent(event, userID, properties)
+	if err != nil {
+		p.API.LogDebug("Error sending user telemetry event", "event", event, "error", err.Error())
+	}
+}
+
+func (p *Plugin) SendDailyTelemetry() {
```

Comment:
This is implemented but doesn't seem to be called from anywhere.

Reply:
You are right, removed unused functions

Reply:
My question was more, should this be called? Why was it added in this PR instead of working with telemetry in a separate PR?

Reply:
☝️

Reply:
@fmartingr 
- Some parts of the codes have been referenced from the stale PR on this plugin only.
- We decided to add telemetry because it was not much of a task.
- If it is required we can move this to a new PR.